### PR TITLE
Update gitignore

### PR DESCRIPTION
**Pull request by serhii73**
Fix - some files from .idea/ want in the repo

### CODE REVIEW

**Comment by serhii73** [comment]
@webknjaz updated `.gitignore`, because some files from .idea/ not ignored.

**Comment by serhii73** [comment]
@webknjaz sorry? why we can't merge it?

File: .gitignore [hunk 1]
```diff
@@ -10,74 +7,6 @@ local_settings.py
...
-# *.iml
-# modules.xml
-# .idea/misc.xml
-# *.ipr
-
-# Sonarlint plugin
-.idea/sonarlint
-
 
-# End of https://www.gitignore.io/api/django,pycharm
```
Comment (webknjaz) [inline .gitignore]: I don't like that you remove this stuff making it harder to update this stuff from upstream.

**Comment by serhii73** [comment]
@webknjaz Corrected, as you said.

**Comment by webknjaz** [comment]
No, you did not rebase.

**Comment by webknjaz** [comment]
And have some PR description, which relates to actual changes

**Comment by serhii73** [comment]
@webknjaz Now, all right?

**Comment by webknjaz** [comment]
No, I still see merge commit and unrelated commits.

**Comment by serhii73** [comment]
@webknjaz And now?

**Comment by webknjaz** [comment]
Please squash those into one commit, because it's all one atomic change.

**Comment by serhii73** [comment]
@webknjaz Maybe now we can to merge in master?

**Comment by serhii73** [comment]
Sorry, @webknjaz can you tell what commits are we talking about?

**Comment by webknjaz** [comment]
https://github.com/kpi-web-guild/django-girls-blog-serhii73/pull/33/commits

**Comment by serhii73** [comment]
@webknjaz Done =)

**Comment by webknjaz** [comment]
Why do you need so much commits?

**Comment by serhii73** [comment]
@webknjaz I think I did `git pull --rebase` and it commits from the master.

**Comment by webknjaz** [comment]
Then something went wrong. Collapse all commits into one.

**Comment by serhii73** [comment]
@webknjaz Done.

**Comment by webknjaz** [comment]
> Merge branch 'master' into update_gitignore

not done.

**Pull request merged**